plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py

```diff
@@ -96,6 +94,8 @@ def __init__(self, column_name):
         self._column_name = column_name
 
     def to_html(self, df: "pd.DataFrame") -> str:
+        import plotly.express as px
```

Comment:
This means we need plotly as a hard dependency on the deck plugin, right?

Reply:
nope, this means if we import `BoxRenderer` and use it in our task, this will be a required dependency